### PR TITLE
Update LiveSplit.AutoSplitters.xml

### PR DESCRIPTION
**Pull request by Lox-r**
If you are adding or modifying an Auto Splitter, please fill out this checklist:
- [x] My Auto Splitter does not contain any malicious functionality and I'm entirely responsible for any damages myself. (Any kind of abuse in an Auto Splitter will result in an immediate ban.)
- [x] I am not replacing an existing Auto Splitter by a different author, or I have received permission from the author to replace the existing Auto Splitter.
- [x] The Auto Splitter has an open source license that allows anyone to fork and host it.

### CODE REVIEW

File: LiveSplit.AutoSplitters.xml [hunk 1]
```diff
@@ -1,5 +1,15 @@
...
+     <AutoSplitter>
+        <Games>
+            <Game>There Is No Game : Wrong Dimension</Game>
+        </Games>
+        <URLs>
+            <URL>https://raw.githubusercontent.com/Lox-r/Autosplitters/refs/heads/main/TINGWD/ThereisNoGameWrongDimension</URL>
+            <URL>https://github.com/just-ero/asl-help/raw/76750096863b03e7ebfdf748aef6d5d9464176a4/lib/asl-help</URL>
+        </URLs>
+        <Type>Script</Type>
+        <Description>Autostart and Autosplit by Lox</Description>
```
Comment (wooferzfg) [inline LiveSplit.AutoSplitters.xml:12]: You need a `</AutoSplitter>` tag

File: LiveSplit.AutoSplitters.xml [hunk 2]
```diff
@@ -1,5 +1,15 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <AutoSplitters>
+     <AutoSplitter>
+        <Games>
+            <Game>There Is No Game : Wrong Dimension</Game>
+        </Games>
+        <URLs>
+            <URL>https://raw.githubusercontent.com/Lox-r/Autosplitters/refs/heads/main/TINGWD/ThereisNoGameWrongDimension</URL>
```
Comment (just-ero) [inline LiveSplit.AutoSplitters.xml:8]: This should really have an `.asl` extension.

**Comment by Lox-r** [comment]
uhhh second time I got no clue what to do, sorry. First time niko was tellin me what to do

**Comment by Lox-r** [comment]
If I do another commit ye will just merge them right?

**Comment by wooferzfg** [comment]
Yes, you can add commits to your branch

**Comment by Lox-r** [comment]
yea sorry for being a pain

**Comment by just-ero** [comment]
Duplicate of #2061, I guess...?

**Pull request closed without merging**